site/src/Main.tsx

```diff
@@ -3,6 +3,8 @@ import { createRoot } from "react-dom/client"
 import { Interpreter } from "xstate"
 import { App } from "./app"
 
+import "./i18n"
```

Comment:
❓ Why do we import this here? Is that to initialize translations or something?

Reply:
Yup, it's part of the [setup](https://react.i18next.com/guides/quick-start)!